lib/Runtime/Base/WindowsGlobalizationAdapter.cpp

```diff
@@ -829,6 +718,7 @@ if (this->object) \
         return ResolveLocaleLookup(scriptContext, locale, resolved);
     }
 
+    // REVIEW (doilij): Return type of this function doesn't match the value returned. Should probably explicitly cast at the call site if needed.
```

Comment:
I changed the return type of getDefaultLocale to be `size_t` in my pull, which is fine because its only used by immediately checking it against 0

Reply:
Okay. Sounds like I'll probably need to resolve that when I rebase.